nodejs/eks/examples/examples_test.go

```diff
@@ -107,6 +107,28 @@ func Test_Examples(t *testing.T) {
 				)
 			},
 		},
+		{
+			Dir: path.Join(cwd, "storage-classes"),
+			Config: map[string]string{
+				"aws:region": region,
+			},
+			Dependencies: []string{
+				"@pulumi/eks",
+			},
+			// Per suggestion in https://git.io/fjVYi due to
+			// https://github.com/pulumi/pulumi/issues/2433
```

Comment:
Perhaps use our standard `TODO` format to make this follow up more greppable?

Reply:
Done